packages/@lwc/rollup-plugin/README.md

```diff
@@ -35,3 +35,4 @@ export default {
 -   `enableLightningWebSecurityTransforms` (type: `boolean`, default: `false`) - The configuration to pass to `@lwc/compiler`.
 -   `enableLwcSpread` (type: `boolean`, default: `false`) - The configuration to pass to the `@lwc/template-compiler`.
 -   `disableSyntheticShadowSupport` (type: `boolean`, default: `false`) - Set to true if synthetic shadow DOM support is not needed, which can result in smaller output.
+-   `apiVersion` (type: `number`, default: `undefined`) - Set to a valid API version such as 58, 59, etc. This will be overriden if the component itself overrides the version with a `*.js-meta.xml` file.
```

Comment:
Already mentioned above, but since `58.0 === 58`, and since I don't think Salesforce will ever have a `.5` version, I'm not too concerned about this. In any case, the TypeScript type is just `number`, so someone can pass in `58.0` if they like.